Realm/ObjectServerTests/RLMObjectServerTests.m

```diff
@@ -582,4 +582,865 @@ - (void)testLogBackInReopenRealmDownload {
     }
 }
 
+- (void)testPermissionChangeForRealmAndUserGrantAdministrativeAccess {
```

Comment:
From a cursory look, it seems there is a lot of copy/paste code in these tests. Can this be refactored?
Also adding a single code comment before each test section would improve readability.